diamond/2.1.9/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:jammy as app
+
+ARG DIAMOND_VER="2.1.9"
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="DIAMOND"
+LABEL software.version="${DIAMOND_VER}"
+LABEL description="Accelerated BLAST compatible local sequence aligner."
+LABEL website="https://github.com/bbuchfink/diamond"
+LABEL license="https://github.com/bbuchfink/diamond/blob/master/LICENSE"
+LABEL maintainer="Kutluhan Incekara"
+LABEL maintainer.email="kutluhan.incekara@ct.gov"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ncbi-blast+ &&\
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+RUN wget --no-check-certificate https://github.com/bbuchfink/diamond/releases/download/v${DIAMOND_VER}/diamond-linux64.tar.gz &&\
+    tar -C /usr/local/bin -xvf diamond-linux64.tar.gz && \
+    rm diamond-linux64.tar.gz
```

Comment:
I'm making notes for my future self:

The reason there is no cd $dir && make and everything is because the only file in diamond-linux64.tar.gz is the diamond executable.